### PR TITLE
Complete the getting-started documentation after the build-system upd…

### PR DESCRIPTION
**Pull request by robertsipka**
…ate.

JerryScript-DCO-1.0-Signed-off-by: Robert Sipka rsipka.uszeged@partner.samsung.com

### CODE REVIEW

File: docs/01.GETTING-STARTED.md [hunk 1]
```diff
@@ -21,51 +21,128 @@ To make our scripts run correctly, several shell utilities should be available o
...
+python tools/build.py --feature=full|compact|minimal
+```
+
+##### Use (jerry|compiler-default|external libc) libc:
+
+The default libc is jerry-libc, but you can use compiler-default libc or an external libc:
+- compiler-default libc:
+
+```bash
+python tools/build.py --jerry-libc=off --compiler-default-libc=on
```
Comment (zherczeg) [inline docs/01.GETTING-STARTED.md]: What happens if --jerry-libc=off and --compiler-default-libc=off? Wouldn't be better if disabling jerry libc would enable compiler libc automatically?

File: docs/01.GETTING-STARTED.md [hunk 2]
```diff
@@ -21,51 +21,128 @@ To make our scripts run correctly, several shell utilities should be available o
...
+```
+
+- external libc:
+
+```bash
+python tools/build.py --jerry-libc=off --compiler-default-libc=off --compile-flag="-I/path/to/libc/include"
+```
```
Comment (zherczeg) [inline docs/01.GETTING-STARTED.md]: Oh this is also possible. Disregard my comment above.

**Comment by zherczeg** [comment]
LGTM

**Comment by LaszloLango** [comment]
LGTM

**Comment by robertsipka** [comment]
I've rebased it with master.

**Pull request merged**